src/pages/projectSettings/tabs/NotificationsTab/types.ts

```diff
@@ -1,42 +1,61 @@
+import { Subset, Unpacked } from "types/utils";
 import { ProjectType } from "../utils/types";
 
+export type Notification = Subset<{
+  notificationSelect: string;
+  jiraCommentInput: string;
+  slackInput: string;
+  emailInput: string;
+  jiraIssueInput: {
+    projectInput: string;
+    issueInput: string;
+  };
+  webhookInput: {
+    urlInput: string;
+    secretInput: string;
+    httpHeaders: { [key: string]: string }[];
+  };
+}>;
+
 export interface FormState {
   buildBreakSettings: {
     notifyOnBuildFailure: boolean | null;
   };
-  subscriptions: Array<{
-    subscriptionData: {
-      event: {
-        eventSelect: string;
-        extraFields: { [key: string]: string };
-        regexSelector: { [key: string]: string }[];
-      };
-      notification: {
-        notificationSelect: string;
-        jiraCommentInput: string;
-        slackInput: string;
-        emailInput: string;
-        jiraIssueInput: {
-          projectInput: string;
-          issueInput: string;
-        };
-        webhookInput: {
-          urlInput: string;
-          secretInput: string;
-          httpHeaders: { [key: string]: string }[];
+  subscriptions: Array<
+    Subset<{
+      subscriptionData: {
+        event: {
+          eventSelect: string;
+          extraFields: { [key: string]: string };
```

Comment:
[nit] Could use `FormExtraFields` here (or delete it)
```suggestion
          extraFields: FormExtraFields;
```